extensions/resteasy-common/deployment/src/main/java/io/quarkus/resteasy/common/deployment/ResteasyDotNames.java

```diff
@@ -39,4 +45,24 @@
     public static final DotName RESTEASY_MATRIX_PARAM = DotName
             .createSimple(org.jboss.resteasy.annotations.jaxrs.MatrixParam.class.getName());
 
+    //types ignored for reflection used by resteasy and smallrye rest client.
+    public static final Set<DotName> TYPES_IGNORED_FOR_REFLECTION = new HashSet<>(Arrays.asList(
+            // javax.json
+            DotName.createSimple("javax.json.JsonObject"),
+            DotName.createSimple("javax.json.JsonArray"),
+
+            // JAX-RS
+            DotName.createSimple(Response.class.getName()),
+            DotName.createSimple(AsyncResponse.class.getName()),
```

Comment:
I know it was preexisting but could you use the class names as strings instead? There's no need to load the classes if they are not used (and they might as well not be).

Reply:
Fixed